flow/util/makeIssue.sh

```diff
@@ -66,6 +66,7 @@ ISSUE_CP_FILES+="${ISSUE_CP_FILES_PLATFORM} \
 
 echo "Creating ${RUN_ME_SCRIPT} script"
 echo "#!/usr/bin/env bash"                     >  ${RUN_ME_SCRIPT}
+echo "export FLOW_HOME=\${FLOW_HOME:-\$(pwd)}" >> ${RUN_ME_SCRIPT}
```

Comment:
In what scenario is this helpful?

Reply:
If you untar and execute ./run*.sh, then FLOW_HOME isn't defined.

Reply:
No script uses FLOW_HOME so I'm unclear why this matters.

Reply:
What script uses FLOW_HOME?

Reply:
vars*.sh

Reply:
```
% grep FLOW_HOME vars-gcd-nangate45-base.sh 
export CURDIR="${FLOW_HOME}"
export FLOW_HOME="."
export LSORACLE_PLUGIN="${FLOW_HOME}/../tools/install/yosys/share/yosys/plugin/oracle.so"
export LSORACLE_KAHYPAR_CONFIG="${FLOW_HOME}/../tools/install/LSOracle/share/lsoracle/test.ini"
```